tests/validation/tests/v3_api/test_kdm_changes.py

```diff
@@ -27,6 +28,8 @@
 CLUSTER_LIST = []
 NODE_COUNT_KDM_CLUSTER = \
     int(os.environ.get("RANCHER_NODE_COUNT_KDM_CLUSTER", 4))
+DNS = \
+    ast.literal_eval(os.environ.get('RANCHER_DNS_PROVIDER', "False"))
```

Comment:
Can we have the RANCHER_DNS_PROVIDER and DNS renamed to RANCHER_DNS_PROVIDER_MATRIX and DNS_MATRIX  to better reflect what they are used for ?

Reply:
Done